taxi/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.conf import settings
+from django.conf.urls.static import static
+from django.contrib import admin
+from django.urls import path
+
+from taxi import views
+
+urlpatterns = [
+    path("", views.index, name="index"),
+] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)

```

Comment:
would make sense to have this line in `taxi_service` since it is project-wide setting